### PR TITLE
UPPSF-1181 Added runbook

### PR DESCRIPTION
**Pull request by powerslider**
# Description

## What

- Added runbook
- Now using circleCI orb
- Fixed missing unit-tests string escaping

## Scope and particulars of this PR (Please tick all that apply)

- [x] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [x] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: runbooks/internal-concordances.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- dimitar.terziev
+
+## Host Platform
+AWS
+
+## Architecture
+Helm chart deployed in Kubernetes. Аccepts a list of concept IDs, concords them, and returns them as an array of UPP concepts.
+
+## Contains Personal Data
+Yes
```
Comment (karsov) [inline runbooks/internal-concordances.md:39]: I believe this is not true.

File: runbooks/internal-concordances.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
...e Light&quot;, &quot;Helvetica Neue&quot;, Arial, sans-serif; font-size: 15px; background-color: rgb(255, 255, 255);"><tbody><tr><td style="line-height: 1.5; padding: 10px 6px; border-bottom-color: rgb(238, 238, 238);">Cluster Health Check</td><td style="line-height: 1.5; padding: 10px 6px; border-bottom-color: rgb(238, 238, 238);"><p><a href="https://upp-prod-delivery-eu.ft.com/__health" target="_blank" style="color: rgb(38, 116, 122);">Prod-EU</a></p><p><a href="https://upp-prod-delivery-us.ft.com/__health" target="_blank" style="color: rgb(38, 116, 122);">Prod-US</a></p></td></tr><tr><td style="line-height: 1.5; padding: 10px 6px; border-bottom-color: rgb(238, 238, 238);">Service Health Check</td><td style="line-height: 1.5; padding: 10px 6px; border-bottom-color: rgb(238, 238, 238);"><p>Cluster aggregate healthchecks are located here:</p><p><a href="https://upp-prod-delivery-eu.ft.com/__health" target="_blank">Prod-EU</a></p><p><a href="https://upp-prod-delivery-us.ft.com/__health" target="_blank">Prod-US</a></p><p>Key "internal-concordances" into the search bar, and click the links to find individual system healthchecks&nbsp;&nbsp;&nbsp;&nbsp;</p></td></tr></tbody></table><br>
```
Comment (karsov) [inline runbooks/internal-concordances.md:82]: This is not the standard content we use for the Monitoring section.

File: runbooks/internal-concordances.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+## First Line Troubleshooting
+<div style="">
```
Comment (karsov) [inline runbooks/internal-concordances.md:85]: This is not the standard content we use for the First Line Troubleshooting section.

**Review by karsov — CHANGES_REQUESTED** [review]
When fixing the tests it would be more beneficial if we were checking for a `url.Error` and not simply fixing the error strings (which may change in the next Go version)

**Review by ivan-p-nikolov — CHANGES_REQUESTED** [review]

File: runbooks/runbook.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+content
+
+## Supported By
+
+content
+
+## Known About By
+
+- elitsa.pavlova
+- kalin.arsov
```
Comment (ivan-p-nikolov) [inline runbooks/runbook.md:32]: Should we stop adding @karsov to runbooks if he is having problems with "managing to many systems"?

**Review by ivan-p-nikolov — APPROVED** [review]

**Review by karsov — APPROVED** [review]

File: runbooks/runbook.md (hunk 4 above)
Comment (karsov) [reply runbooks/runbook.md:32]: And I can use: https://apigateway.in.ft.com/key-form/developer

**Pull request merged**